Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.10.1
+WORKDIR /go/src/github.com/netlify/twickr
+COPY . /go/src/github.com/netlify/twickr/
+RUN go get && go build
+
+FROM alpine:3.7
+RUN apk add --no-cache ca-certificates
+RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
```

Comment:
I don't think this is necessary. The only reason `gotrue` had it was because of a library that was using `cgo`.

Reply:
I had to do it because the binary was looking for that library. I guess it's related to not doing an static build